src/EA.Weee.Web/Areas/AatfReturn/Views/SelectReportOptionsDeselect/Index.cshtml

```diff
@@ -16,27 +16,30 @@
                 <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
                 <strong class="govuk-warning-text__text">
                     <span class="govuk-warning-text__assistive">Warning</span>
-                    Any data entered against these options will be deleted
+                    Any data entered against the options in the list will be deleted
                 </strong>
             </div>
         </div>
     </div>
-    
+
     <div class="govuk-!-padding-bottom-5"></div>
-    @foreach (var deselectedOption in Model.ReportOnQuestions.Where(r => r.DeSelected))
-    {
-        <p class="govuk-body">
-            @if (deselectedOption.AlternativeDescription != null)
-            {
-                @deselectedOption.AlternativeDescription
-            }
-            else
+    <div class="govuk-body">
+        <ul>
+            @foreach (var deselectedOption in Model.ReportOnQuestions.Where(r => r.DeSelected))
             {
-                @deselectedOption.Title
+                <li style="padding-top: 10px; list-style-type: none;">
```

Comment:
I think you could use ol here... as I think that has list-style-type: none defined. Also for the padding you could use govuk-!-padding-top-3 as that has defined padding.